### PR TITLE
Fixed bootstrapping with reserved keywords in role name, deprecate --default-database and -user

### PR DESCRIPTION
**Pull request by tailhook**
Discussed in #1879

### CODE REVIEW

**Review by 1st1 — APPROVED** [review]
lgtm, but let @elprans looks at this too

File: edb/server/cluster.py [hunk 1]
```diff
@@ -256,6 +256,8 @@ def _admin_query(self, query):
         conn_args = self.get_connect_args().copy()
         conn_args['host'] = str(self._runstate_dir)
         conn_args['admin'] = True
+        conn_args['user'] = 'edgedb'
```
Comment (elprans) [inline edb/server/cluster.py:259]: use `edbdef` constants here, please

Comment (tailhook) [reply edb/server/cluster.py]: Oh, yeah. Sorry! Fixed

**Review by elprans — APPROVED** [review]

**Comment by elprans** [comment]
Tests are hanging for some reason. I suspect it might be `server_ops` tests

**Comment by tailhook** [comment]
@elprans please take another look. I'm not sure about [this](https://github.com/edgedb/edgedb/pull/1883/files#diff-58b7a36dbe970d63a11f59c04b1ed86f0a40cfcecbc8890d1f4ba90d19b4284bR206-R209)

File: edb/server/main.py [hunk 2]
```diff
@@ -201,8 +202,11 @@ def _run_server(cluster, args: ServerConfig,
     if bootstrap_script_text is not None:
         bootstrap_script = server.StartupScript(
             text=bootstrap_script_text,
-            database=args.default_database,
-            user=args.default_database_user,
+            database=args.default_database or edgedb_defines.EDGEDB_SUPERUSER,
+            user=(
+                args.default_database_user or
+                edgedb_defines.EDGEDB_SUPERUSER_DB
```
Comment (elprans) [inline edb/server/main.py:208]: This is fine, but you mixed the `_DB` and `USER` constants.

**Comment by elprans** [comment]
Please don't forget to squash fixup commits

**Comment by tailhook** [comment]
> Please don't forget to squash fixup commits

Sorry. Too used to squash by default :(

**Pull request merged**